src/features/CourtCaseList/CourtCaseList.tsx

```diff
@@ -34,14 +35,20 @@ const CourtCaseList: React.FC<Props> = ({ courtCases, order = "asc", currentUser
   const classes = useStyles()
   const { basePath, query } = useRouter()
 
-  deleteQueryParamsByName(["unlockException", "unlockTrigger"], query)
+  let searchParams = new URLSearchParams(encode(query))
+  searchParams = deleteQueryParamsByName(["unlockException", "unlockTrigger"], searchParams)
 
   const orderByParams = (orderBy: string) => `${basePath}/?${new URLSearchParams({ ...query, orderBy, order })}`
   const caseDetailsPath = (id: number) => `${basePath}/court-cases/${id}`
-  const unlockExceptionPath = (unlockException: string) =>
-    `${basePath}/?${new URLSearchParams({ ...query, unlockException })}`
-  const unlockTriggerPath = (unlockTrigger: string) =>
-    `${basePath}/?${new URLSearchParams({ ...query, unlockTrigger })}`
+  const unlockExceptionPath = (unlockException: string) => {
```

Comment:
`unlockExceptionPath` and `unlockTriggerPath` seem to be identical apart from the fact that one of them uses `unlockException` string & the other `unlockTrigger` string. Do you think it would make sense to have this as 1 function?

Reply:
refactored into 1 function